verification/src/config.rs

```diff
@@ -29,13 +27,20 @@ impl Default for ServerConfiguration {
 
 #[derive(Deserialize, Clone)]
 #[serde(default)]
-pub struct VerifierConfiguration {
+pub struct SolidityConfiguration {
     pub enabled: bool,
+    pub compilers_list_url: Url,
 }
 
-impl Default for VerifierConfiguration {
+impl Default for SolidityConfiguration {
     fn default() -> Self {
-        Self { enabled: true }
+        Self {
+            compilers_list_url: Url::try_from(
+                "https://raw.githubusercontent.com/blockscout/solc-bin/main/list.json",
```

Comment:
Let's maybe define default link to be dependent on the OS user runs the service. And use ethereum/solc-bin list.json for MacOS and Windows

Reply:
Something like 
```
#[cfg(target_os = "linux")]
const DEFAULT_COMPILER_LIST: &'static str = "https://raw.githubusercontent.com/blockscout/solc-bin/main/list.json";
#[cfg(target_os = "macos")]
const DEFAULT_COMPILER_LIST: &'static str = "https://solc-bin.ethereum.org/macosx-amd64/list.json";
#[cfg(target_os = "windows")]
const DEFAULT_COMPILER_LIST: &'static str = "https://solc-bin.ethereum.org/windows-amd64/list.json";

impl Default for SolidityConfiguration {
    fn default() -> Self {
        Self {
            compilers_list_url: Url::try_from(
                DEFAULT_COMPILER_LIST,
            )
            .expect("valid url"),
            enabled: true,
        }
    }
}
```
That way we may create valid versions of list.json for all main OS

Reply:
okay, let me remove `test_config.toml` for now we don't need it